index.js

```diff
@@ -15,9 +15,71 @@ document.addEventListener('DOMContentLoaded', () => {
       displayPlanPrice: '$29.90', }));
   }, 5000);
 
+  const id = faker.string.uuid();
   // eslint-disable-next-line no-unused-expressions
-  !localStorage.getItem('__cid') && localStorage.setItem('__cid', faker.datatype.uuid());
-  ReactBotClient({ root: document.querySelector('#root') });
+  !localStorage.getItem('__cid') && localStorage.setItem('__cid', id);
+
+  // DEV Note: will be received by Razvigor
+  const initialConfig = {
+    meta: {
+      cid: id,
+      pd: {
+        subscriberBillingFrequency: 'Billed every {1} month(s)',
+        oneTimer: 'Total price to be charged',
+        billingFrequencyTmsg: '',
+      },
+      systemType: 'test',
+      marketing: {
+        utmParams: { utm_chat: 'db-sales-nopresetquestions' },
+        lastUtmParams: { utm_chat: 'db-sales-nopresetquestions' },
+        screen: {
+          width: 2084,
+          height: 1608,
+        }
+      },
+      eid: '23c7cdcf-4d90-4ea1-aab0-c73f8426dc1d_PageView_1694521420366',
+    },
+    app: {
+      aiProfile: {
+        name: 'Meal Mentor',
+        role: 'AI-powered nutritionist',
+        imgSrc: 'https://storage.1forfit.com/4oZrkOwbOQcSIGJopBG5qsf0CmBbVDKDqflzEkXq.jpg',
+        displayInStream: true,
+        welcome: 'Welcome to our live support. We\'re here to understand your requirements and suggest the best Keto diet suited for you.',
+        // initialMessage: 'Hi, {I am Meal Mentor}. I will help you to find the right meal plan for you. [yes|no|continue]',
+        initialMessage: 'Do you want to lose weight? [Yes|No]',
+      },
+      chatUrl: 'https://yourketo.ngrok.io',
+      themeId: 'dark',
+      translations: {
```

Comment:
Can we move part of the configuration in a config file, just for better code arrangement. I think that at least the translations can be in some translation config.